### PR TITLE
master-lps-104195

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-109153
https://issues.liferay.com/browse/LPS-109152
https://issues.liferay.com/browse/LPS-109151

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/brianchandotcom/liferay-portal/pull/85263<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/d8d03f7c40ff~1...d8d03f7c40ff)

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotMemberships.testcase [hunk 1]
```diff
@@ -0,0 +1,142 @@
+@component-name = "portal-document-management"
+definition {
+
+	property portal.release = "false";
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotMemberships.testcase]: @Jeremy-Ch Can you file a test improvement to change this to "true?" I forgot to change it

**Pull request closed without merging**